app/views/extension/index.phtml

```diff
@@ -4,12 +4,10 @@
 	<a href="<?= _url('index', 'index') ?>"><?= _t('gen.action.back_to_rss_feeds') ?></a>
 
 	<h1><?= _t('admin.extensions.title') ?></h1>
-
+    <h2><?= _t('admin.extensions.system') ?></h2>
```

Comment:
Please mind the white-space indenting and use the same than the rest of the file, i.e. tabs

Reply:
sorry. Now it is corrected